src/action/bandaidset.rs

```diff
@@ -0,0 +1,494 @@
+//! A set of bandaids, which refer the changes required to apply one single suggestion.
+
+use std::convert::TryFrom;
+
+use anyhow::{anyhow, Error, Result};
+use log::trace;
+
+use crate::suggestion::Suggestion;
+use crate::{LineColumn, Span};
+use crate::CheckableChunk;
+use crate::{CommentVariant, LineColumn, Replacement, Span};
+
+use super::BandAid;
+
+/// A set of `BandAids` for an accepted suggestion.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct FirstAidKit {
+    /// All `Bandaids` in this kit constructed from the replacement of a suggestion,
+    /// each bandaid covers at most one complete line
+    pub bandaids: Vec<BandAid>,
+}
+
+impl Default for FirstAidKit {
```

Comment:
Can't we just derive this?

Reply:
As mentioned somewhere, `FirstAidKit` is dead. I'll remove this file as a whole.